---
title: "multiStringTie: a multi-sample transcriptome assembler"
collection: researches
type: "PhD Student"
permalink: /researches/2022-10-20-multistringtie
venue: "<a href='https://bits.iis.sinica.edu.tw/?id=1' target='_blank' style='color: inherit;'>Center of Computational Biology, Johns Hopkins University</a>"
start_date: 2021-10-01
end_date: "Present"
location: "Baltimore, MD"
status: '<a style="text-decoration:none; color:orange;">In progress <i class="fa fa-spinner" aria-hidden="true"></i></a>'
image: "/images/multistringtie.png"
# superviser: '<a href="/references/Huai-Kuang_Tsai" style="font-size: 12px; text-decoration:none; color:#4A4F53; border-style: solid; border-color:#bfcae3; border-radius: 10px; background-color: #bfcae3;" target="_blank">&nbsp; Huai-Kuang Tsai &nbsp;</a>'
superviser_clean:
  - "Steven Salzberg"
# research_position: "RA_IIS_AS"
research_clean: "multistringtie"
---

I am developing a multi-sample assembler upon <a href="https://ccb.jhu.edu/software/stringtie/index.shtml" target="_blank" >StringTie</a>, the state-of-the-art transcriptome assembler, aiming to construct a more precise gene catalog by taking advantage of the complementarity and redundancy of multiple samples.
multiStringTie infers transcripts from “universal splicing graphs”, multi-samplewise-refined directed splicing graphs representing
gene loci, to achieve better sensitivity-precision tradeoff.

<iframe src="https://docs.google.com/presentation/d/e/2PACX-1vSy6Rd_CNn10ca8hBK4G1chFC80wVEc5kLEVDuIa56MsQnUrN1pvLK2lmzUxDq3gg/embed?start=false&loop=false&delayms=3000" frameborder="0" width="100%" height="500" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>

<iframe src="https://docs.google.com/presentation/d/e/2PACX-1vRtTIfKhJ5LsoDQor9atEhLqPiWfvqu50T-EO4_mtf9DQOJue8rsmAoharKeBfqdg/embed?start=false&loop=false&delayms=3000" frameborder="0" width="100%" height="500" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
---
