---
title: "sangeranalyseR: simple and interactive processing of Sanger sequencing Data in R"
collection: researches
type: "Research assistant"
permalink: /researches/2019-10-01-sangeranalyseR
venue: "<a href='http://www.robertlanfear.com/' target='_blank' style='color: inherit;'>Lanfear Lab, Division of Ecology and Evolution, Research School of Biology, Australian National University</a>"
start_date: 2019-09-01
end_date: 2020-01-01
location: "Canberra, Australia"
status: '<a style="text-decoration:none; color:green;">Published <i class="fa fa-check-circle" aria-hidden="true"></i></a> &nbsp; <a href="https://academic.oup.com/gbe/advance-article/doi/10.1093/gbe/evab028/6137837?guestAccessKey=a28b32d6-ffab-41f2-8132-9c2dd28b99fe
" target="_blank"><i style="font-size: 12px; color:green">Genome Biology and Evolution</i></a>'
image: "/images/sangeranalyseR_cover.png"
image_bioconductor: "Yes"
superviser: '<a href="/references/Robert_Lanfear" style="font-size: 12px; text-decoration:none; color:#4A4F53; border-style: solid; border-color:#e3bfbf; border-radius: 10px; background-color: #e3bfbf;" target="_blank">&nbsp; Robert Lanfear &nbsp;</a>'
superviser_clean:
  - "Robert Lanfear"
research_position: "RA_RSB_ANU"
research_clean: "sangeranalyseR"
---

I developed sangeranalyseR while working as a research assistant at the <a target="_blank"  href="http://www.robertlanfear.com/" style="color:#4A4F53">Molecular Evolution and Phylogenetics Lab</a>, led by Prof. <a target="_blank"  href="https://biology.anu.edu.au/people/academics/robert-lanfear" style="color:#4A4F53">Robert Lanfear</a> during my exchange at the Australian National University. sangeranalyseR is now available on Bioconductor 3.12.


This package builds on sangerseqR to allow users to create contigs from collections of Sanger sequencing reads. It provides a wide range of options for a number of commonly-performed actions including read trimming, detecting secondary peaks, and detecting indels using a reference sequence. All parameters can be adjusted interactively either in R or in the associated Shiny applications. There is extensive online documentation, and the package can outputs detailed HTML reports, including chromatograms.

<figure style="text-align: center; display: inline-block;margin-top:-1px">
  <img src="/images/sangeranalyseR_figure_1.png" style="width: 90%; display: block;margin-left: auto;margin-right: auto;">
  <figcaption style="text-align: center;">The workflow of sangeranalyseR.
  </figcaption>
</figure>

---

<h2 style="color: #000f70; margin-left:-30px" > <i class="fas fa-dot-circle" style="font-size:18px;"></i> &nbsp;&nbsp;Source code & Documentation </h2>

<div>
  <div style="margin-left:30px">
    <iframe src="https://bioconductor.riken.jp/packages/3.12/bioc/html/sangeranalyseR.html" width="75%" height="300" style="display: block;" title="W3Schools Free Online Web Tutorials"></iframe>
  </div>
  <ul>
    <li><a href="https://bioconductor.org/packages/sangeranalyseR" target="_blank">sangeranalyseR <b>Bioconductor</b></a></li>
    <li><a href="https://github.com/roblanf/sangeranalyseR" target="_blank">sangeranalyseR <b>GitHub repository</b></a></li>
    <li><a href="https://sangeranalyser.readthedocs.io/" target="_blank">sangeranalyseR <b>Documentation</b></a></li>
    <li><a href="https://bioconductor.org/packages/devel/bioc/vignettes/sangeranalyseR/inst/doc/sangeranalyseR.html" target="_blank">sangeranalyseR <b>Vignettes</b></a></li>
  </ul>
</div>

---
