---
title: "RNASeqR: an R package for automated two-group RNA-Seq
analysis workflow"
collection: publications
permalink: /publication/2019_12_04-RNASeqR
excerpt: '<small>This R package is designed for case-control RNA-Seq analysis (two-group). There are six steps: "RNASeqRParam S4 Object Creation", "Environment Setup", "Quality Assessment", "Reads Alignment & Quantification", "Gene-level Differential Analyses" and "Functional Analyses". Each step corresponds to a function in this package. After running functions in order, a basic RNASeq analysis would be done easily.</small>'
date: 2019-12-04
venue: '<b>IEEE/ACM Transactions on Computational Biology and Bioinformatics (TCBB)</b>'
paperurl: 'https://ieeexplore.ieee.org/document/8918337'
citation: '<br><b style="color:#ad0000">K.H. Chao</b>, Y.W. Hsiao, Y.F. Lee, C.Y. Lee, L.C. Lai, M.H. Tsai, T.P. Lu, and E.Y. Chuang* (2019). "RNASeqR: an R package for automated two-group RNA-Seq analysis workflow" in <b><i>IEEE/ACM Transactions on Computational Biology and Bioinformatics (TCBB)</i></b>. doi: doi.org/10.1109/TCBB.2019.2956708.'
SJR: '<a href="https://www.scimagojr.com/journalsearch.php?q=17971&amp;tip=sid&amp;exact=no" title="SCImago Journal &amp; Country Rank"><img border="0" src="https://www.scimagojr.com/journal_img.php?id=17971" style="width:235px; height: 250px;object-fit: cover;display: inline; margin-top:20px;" alt="SCImago Journal &amp; Country Rank"  /></a>'
altmetric: "<div class='altmetric-embed' data-badge-type='1' data-doi='10.1101/641324' style='display:inline;'></div>"
license: '<a href="https://opensource.org/licenses/Artistic-2.0" target="_blank"><img src="https://img.shields.io/badge/License-Artistic%202.0-0298c3.svg"></a>'
document_status: '<a href="https://www.bioconductor.org/packages/release/bioc/vignettes/RNASeqR/inst/doc/RNASeqR.html" target="_blank"><img src="https://img.shields.io/badge/docs-passing-brightgreen.svg"></a>'
platforms: '<a href="https://www.bioconductor.org/packages/release/bioc/html/RNASeqR.html" target="_blank"><img src="https://img.shields.io/badge/platform-macOS_/Linux-green.svg"></a>'
superviser_clean:
  - "Eric Y. Chuang"
  - "Tzu-Pin Lu"
research_clean: "RNASeqR"
---

<script src="https://kit.fontawesome.com/yourcode.js"></script>


<div clss="row" style="display: flex; column-count: 3;">
  <div class="column">
    <div class='altmetric-embed' data-badge-type='medium-donut' data-doi='10.1101/641324' style="inline; margin-top:10px"></div>
  </div>
  <div class="column">
    <a href="https://www.scimagojr.com/journalsearch.php?q=17971&amp;tip=sid&amp;exact=no" title="SCImago Journal &amp; Country Rank"><img border="0" src="https://www.scimagojr.com/journal_img.php?id=17971" style="width:60%;object-fit: cover;display: inline; margin-left:60px;" alt="SCImago Journal &amp; Country Rank"  /></a>
  </div>
  <div class="column">
  </div>
</div>

<h2 style="color: #000f70"> <i class="fas fa-dot-circle" style="font-size:18px;"></i> &nbsp;&nbsp;Abstract </h2>

<div style="margin-left: 30px">
RNA-Seq analysis has revolutionized researchersâ€™ understanding of the transcriptome in biological research. Assessing the differences in transcriptomic profiles between tissue samples or patient groups enables researchers to explore the underlying biological impact of transcription. RNA-Seq analysis requires multiple processing steps and huge computational capabilities. There are many well-developed R packages for individual steps; however, there are few R/Bioconductor packages that integrate existing software tools into a comprehensive RNA-Seq analysis and provide fundamental end-to-end results in pure R environment so that researchers can quickly and easily get fundamental information in big sequencing data. To address this need, we have developed the open source R/Bioconductor package, RNASeqR. It allows users to run an automated RNA- Seq analysis with only six steps, producing essential tabular and graphical results for further biological interpretation. The features of RNASeqR include: six-step analysis, comprehensive visualization, background execution version, and the integration of both R and command-line software. RNASeqR provides fast, light-weight, and easy-to-run RNA-Seq analysis pipeline in pure R environment. It allows users to efficiently utilize popular software tools, including both R/Bioconductor and command-line tools, without predefining the resources or environments. RNASeqR is freely available for Linux and macOS operating systems from Bioconductor (<a href="https://bioconductor.org/packages/release/bioc/html/RNASeqR.html">https://bioconductor.org/packages/release/bioc/html/RNASeqR.html</a>).
</div>

<h2 style="color: #000f70"> <i class="fas fa-dot-circle" style="font-size:18px;"></i> &nbsp;&nbsp;Conclusion </h2>

<div style="margin-left: 30px">
RNASeqR is a new Bioconductor package providing a six- step automated workflow for two-group comparative RNA-Seq analysis. The core design concept of RNASeqR is that each RNA-Seq analysis step is implemented as an R function in the package, and thus users can perform RNA- Seq analysis instinctively. The main features of RNASeqR include: (i) flexible function options in both an R-interac- tive version and a background version, (ii) comprehensive visualization, (iii) integration of command-line tools into the R environment, (iv) a well-defined file structure, and (v) ease of use. We believe that RNASeqR will assist clinical researchers without significant computational background to obtain useful information from RNA-Seq data in an easy, efficient and reproducible manner.
</div>


<h2 style="color: #000f70"> <i class="fas fa-dot-circle" style="font-size:18px;"></i> &nbsp;&nbsp;Selected Figure </h2>

<div style="margin-left: 30px">
<img src="{{base_path}}/images/RNASeqR_figure_1.png">

<div style = "margin: 30px">
<small>Overview of the full workflow of RNASeqR. There are three parts in this figure: the main workflow (middle), the functions involved in each step (on the left), and the substeps that can be performed within each function (on the right). Aside from the function S4 RNASeqRParam, the remaining functions can be conducted in either the R-interactive version or the background version (by adding the CMD suffix). After correctly running the functions in order, a two-group comparative RNA-Seq data analysis will be done.</small>
</div>
</div>


<h2 style="color: #000f70"> <i class="fas fa-dot-circle" style="font-size:18px;"></i> &nbsp;&nbsp;Source code & Documentation </h2>

<div style="margin-left: 15px">
  <ul>
    <li><a href="https://bioconductor.org/packages/release/bioc/html/RNASeqR.html">RNASeqR <b>Bioconductor</b></a></li>
    <li><a href="https://github.com/Kuanhao-Chao/RNASeqR">RNASeqR <b>GitHub repository</b></a></li>
    <li><a href="https://bioconductor.org/packages/release/bioc/vignettes/RNASeqR/inst/doc/RNASeqR.html">RNASeqR <b>Vignettes</b></a></li>
    <li><a href="https://bioconductor.org/packages/release/data/experiment/html/RNASeqRData.html">RNASeqRData <b>Bioconductor</b></a></li>
  </ul>
</div>


<br>

---

**Contact:**&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[kuanhao.chao@gmail.com](mailto:kuanhao.chao@gmail.com)

**Index Terms:** &nbsp;&nbsp;&nbsp;&nbsp;*RNA-Seq*, *Analysis workflow*, *Pipeline*, *R*, *Bioconductor*, *Transcriptome assembly*, *Differential expression analysis*, *Gene expression*, *Statistical analysis*, *Visualization*

<br>

---
[**>> Download paper here <<**](https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=8918337&isnumber=4359833)
